Resources/public/js/views/fields/ez-media-view.js

```diff
@@ -32,10 +32,25 @@ YUI.add('ez-media-view', function (Y) {
         _variables: function () {
             return {
                 "isAudio": this.get('fieldDefinition').fieldSettings.mediaType === "TYPE_HTML5_AUDIO",
+                "uriWithTimestamp": this._generateUriWithTimestamp(),
             };
         },
 
         /**
+         * Generate the URI with a timestamp a a parameter to avoid browser caching
+         *
+         * @method _generateUriWithTimestamp
+         * @return {String} The URI with a timestamp parameter or "" if field is empty
+         */
+        _generateUriWithTimestamp: function() {
+            if (this._isFieldEmpty()) {
+                return "";
+            }
+
+            return this.get('field').fieldValue.uri + "?t=" + Date.now();
```

Comment:
The version number is not available in this view. But looking for it I found an id that does the trick :smile: 

Following @dpobel 's advice I modified the URL in the template only.